src/Extractor/Common.php

```diff
@@ -39,11 +41,13 @@ public function __construct(array $parameters, array $state, LoggerInterface $lo
 
     protected function createExportAdapter(): ExportAdapter
     {
+        $resultWriter = new DefaultResultWriter($this->state);
         $simpleQueryFactory = new DefaultQueryFactory($this->state);
         return new PdoExportAdapter(
             $this->logger,
             $this->connection,
             $simpleQueryFactory,
+            $resultWriter,
```

Comment:
Result writer sa da nastavovat zvonku, ... to sa menilo uz skor, ale nebolo to tu upravene.